docs/how-to-guides/isis-developer-guides/developing-mac-m1-arm-systems.md

```diff
@@ -0,0 +1,117 @@
+# ISIS Development on the Mac M1 ARM System
+
+*By [Kris Becker](https://github.com/KrisBecker) (see original GitHub post [here](https://github.com/DOI-USGS/ISIS3/issues/5188))*
+
+In a recent ISIS Technical Committee [meeting](https://github.com/planetarysoftware/ISIS_TC/blob/main/meetings/2023-04-13.md), ISIS support for development on the Mac Arm platform was discussed. This document describes my experience with existing methods/procedures used to develop [ISIS](https://github.com/DOI-USGS/ISIS3) on the Mac M1 system.
+
+There are at least two ways to develop ISIS on the M1. The current ISIS configuration can be built on the M1 in the Rosetta Translation Environment using the x86_i386 Intel-based instruction set. Building ISIS natively using the x86_64 ARM instruction set is not yet possible. The instructions to set up both environments is provided here, but the x86_i386 is recommended until all the required dependencies are available in the ARM environment.
+
+Note that my M1 Mac system currently runs Apple macOS Monterey. Here is some additional information regarding the system used to install, configure and build ISIS on the M1:
```

Comment:
We should not make it first person, update this with an [admonition](https://squidfunk.github.io/mkdocs-material/reference/annotations/#in-content-tabs-tab-1) like: 

```markdown
???+ note

    This guide is based on a M1 Mac system running Apple macOS Monterey. Here is some additional information regarding the system used to install, configure and build ISIS on the M1:

    ```
    Hardware Overview
      Model Name:	MacBook Pro
      Model Identifier:	MacBookPro18,2
      Chip:	Apple M1 Max
      Total Number of Cores:	10 (8 performance and 2 efficiency)
      Memory:	32 GB
    Graphics/Display
      Type:	GPU
      Bus:	Built-In
      Total Number of Cores:	24
    System Software Overview:
      System Version:	macOS 12.6.3 (21G419)
      Kernel Version:	Darwin 21.6.0
    ```
```